conf/rubocop_without_rspec.yml

```diff
@@ -231,3 +231,9 @@ Style/RedundantSelfAssignment:
 
 Style/SingleArgumentDig:
   Enabled: false
+
+Style/TrailingCommaInArrayLiteral:
+  EnforcedStyleForMultiline: consistent_comma
+
+Style/TrailingCommaInHashLiteral:
```

Comment:
Should we include `Style/TrailingCommaInArguments` as well?

Reply:
I'm up for it. Rules and patterns seems to be the same so 👍

Reply:
👍 I added a commit here for `Style/TrailingCommaInArguments`

Reply:
We should bring the `Style/TrailingCommaInArguments` cop question back to #rubocop-changes for a broader discussion